packages/zscaler_zia/data_stream/sandbox_report/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,568 @@
+---
+description: Pipeline for processing Sandbox Report logs.
+processors:
+  - set:
+      field: ecs.version
+      tag: set_ecs_version
+      value: 8.11.0
+  - rename:
+      field: message
+      tag: rename_message_to_event_original
+      target_field: event.original
+      ignore_missing: true
+      description: Renames the original `message` field to `event.original` to store a copy of the original message. The `event.original` field is not touched if the document already has one; it may happen when Logstash sends the document.
+      if: ctx.event?.original == null
+  - remove:
+      field: message
+      tag: remove_message
+      ignore_missing: true
+      description: The `message` field is no longer required if the document has an `event.original` field.
+      if: ctx.event?.original != null
+  - json:
+      field: event.original
+      tag: json_event_original
+      target_field: json
+      if: ctx.event?.original != null
+      on_failure:
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - rename:
+      field: json.Summary
+      tag: rename_Summary_to_Full_Details
+      target_field: json.Full Details
+      ignore_missing: true
```

This does not look right. The field in the original doc has a field `Full Details`, not `Summary`.
```suggestion
  - rename:
      field: json.Full Details
      tag: rename_Full_Details_to_summary
      target_field: json.summary
      ignore_missing: true
```
Then all the following map accesses can be simplified to dot and safe-dot accesses.